src/IssueViz/IssueVizPackage.cs

```diff
@@ -33,7 +33,7 @@ namespace SonarLint.VisualStudio.IssueVisualization
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid("4F3D7D24-648B-4F3B-ACB0-B83AFE239210")]
     [ProvideMenuResource("Menus.ctmenu", 1)]
-    [ProvideToolWindow(typeof(IssueVisualizationToolWindow), MultiInstances = false, Transient = true, Style = VsDockStyle.Float, Width = 325, Height = 400)]
+    [ProvideToolWindow(typeof(IssueVisualizationToolWindow), MultiInstances = false, Style = VsDockStyle.Float, Width = 325, Height = 400)]
```

Comment:
I had to remove the Transient.
I don't know why, but it caused the window to stop automatically appearing and disappearing.
I reset the hive, I did a fresh installation on 2017 to make sure that it wasn't caching that the window is closed, but it still didn't work.
I think we need to pick whether we want to document that a goldbar can appear, or remove the auto show/hide. :/